src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj

```diff
@@ -8,7 +8,10 @@
     <Version>1.0.0</Version>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-
+  <PropertyGroup>
+    <ExcludeFromBuild>true</ExcludeFromBuild>
```

Comment:
Is this a new filtering mechanic?

Reply:
Yes this makes it very clear for two occasions, one for swagger to .NET SDK and one where these changes are local to the SDK